packages/firebase_auth/firebase_auth/windows/messages.g.cpp

```diff
@@ -2752,14 +2777,10 @@ void FirebaseAuthHostApi::SetUp(flutter::BinaryMessenger* binary_messenger,
               const auto& email_arg =
                   std::get<std::string>(encodable_email_arg);
               const auto& encodable_action_code_settings_arg = args.at(2);
-              const PigeonActionCodeSettings* action_code_settings_arg =
-                  nullptr;
-              if (!encodable_action_code_settings_arg.IsNull()) {
-                action_code_settings_arg =
-                    &(std::any_cast<const PigeonActionCodeSettings&>(
-                        std::get<CustomEncodableValue>(
-                            encodable_action_code_settings_arg)));
-              }
```

Comment:
I think this should be put back, not related to the modifications

Reply:
Another manual update. We should probably make a script for these.